### ISSUE TITLE
Sourcemaps are unable to be uploaded when publishing a site with Wrangler

### ISSUE THREAD

**Issue by signalnerve** [cloudflare/wrangler#667]
### 🐛 Bug Report

When trying to publish a Gatsby site, the sourcemaps generated for each page are large enough that they make the `publish` step fail. IMO we should ignore the sourcemap files (maybe via a config) or warn on this instead of throwing an error.

```
❯ ~/.cargo/bin/wrangler publish
🌀  Creating namespace for Workers Site "__frontendjobs-closedsource-workers_sites_assets"
Error: File `component---src-pages-post-js-c6d1c3aab5c008b72fa8.js.map-b3fd6703031f027b11dd2dc7e3448fe3838efa53e5c6436c4aa3dd9c721cc7e4` of 3252144 bytes exceeds the maximum value size limit of 2097152 bytes
```

#### Environment

`wrangler` built from master this AM 

#### Steps to reproduce

Run `wrangler publish` on a Workers site with large sourcemaps. Sorry, my project is closed-source, but happy to give access to Wrangler team if needed!

**Comment by ashleygwilliams** [comment]
@signalnerve can you say more about what you'd like to see here?

i could imagine that we could fail on publish with a warning that files are too big and then encourage you to run the command again... skipping the too large files

OR

you could drop a .wranglerignore in the route of your project to ignore files that are too large?

**Comment by gabbifish** [comment]
This is similar to #464

**Comment by ashleymichal** [comment]
the merged issue solved a similar issue, but this one still exists and is being addressed by #720, which closes issue #716
